handlers/register.go

```diff
@@ -100,6 +100,7 @@ func Register(mainRouter *mux.Router, cfg config.Config) {
 	// callback endpoints should not be intercepted by the paymentauth or userauth interceptors, so needs to be it's own subrouter
 	callbackRouter := mainRouter.PathPrefix("/callback").Subrouter()
 	callbackRouter.HandleFunc("/payments/govpay/{payment_id}", HandleGovPayCallback).Methods("GET").Name("handle-govpay-callback")
+	callbackRouter.Handle("/payments/paypal/orders/{payment_id}", HandlePayPalCallback(payPalService)).Methods("GET").Name("handle-paypal-callback")
```

Comment:
I would remove the `orders` in this URL as although it's technically an "order" with PayPal that doesn't really mean anything in our payments service, and as we won't use any other pay functionalities on PayPal we don't need to differentiate what's an order or not. This will also have to be changed [here](https://github.com/companieshouse/payments.api.ch.gov.uk/blob/master/service/paypal.go#L77)

Reply:
This was a suggestion from Kevin Bradwick after reviewing the Low Level Design - his reasoning was to distinguish if we ever needed a callback with PayPal's payments endpoint